app/models/user.rb

```diff
@@ -1,7 +1,42 @@
 class User < ApplicationRecord
+  attr_accessor :remember_token
+
   validates :name, presence: true, length: { maximum: 30 }
   validates :email, presence: true, length: { maximum: 255 }, format: { with: URI::MailTo::EMAIL_REGEXP }, uniqueness: true
 
   has_secure_password
   validates :password, presence: true, length: { minimum: 6 }
```

Comment:
try to keep all the validates together. For example:
validates: 
validates:
validates:

has_secure_password